android/build.gradle

```
@@ -68,13 +68,16 @@ afterEvaluate {
 android {
   compileSdkVersion safeExtGet("compileSdkVersion", 31)
```

With the bump to 17 for the java version we can probably bump the compile version to the latest as well.
```suggestion
  compileSdkVersion safeExtGet("compileSdkVersion", 34)
```